.circleci/config.yml

```diff
@@ -494,7 +495,7 @@ jobs:
           no_output_timeout: 15m
           environment:
             SCAN_PLATFORM: watchOS Simulator
-            SCAN_DEVICE: Apple Watch Series 8 (41mm),OS=9.4
+            SCAN_DEVICE: Apple Watch Series 9 (45mm),OS=10.4
```

Comment:
Not sure why this needs to be specified with `OS=10.4`. Changing it to `(10.4)` makes it fail, I think it might be that this is using `xcodebuild` directly and not `scan`?

Reply:
not sure, but I imagine that might be the case. why is it using xcode build directly but setting the var as `SCAN_DEVICE`?

Reply:
I don't know, might also be a copy paste thing. The lane is doing

```
lane :test_revenuecatui do |options|
    generate_snapshots = ENV["CIRCLECI_TESTS_GENERATE_REVENUECAT_UI_SNAPSHOTS"] == "true"
    platform = ENV['SCAN_PLATFORM'] || 'iOS Simulator'
    destination = ENV['SCAN_DEVICE'] || "iPhone 14,OS=16.4"
    sdk = ENV['BUILD_SDK'] || 'iphonesimulator'
```

so we have control over the names. I might honestly rename them because it is confusing